OpenRA.Game/Platform.cs

```diff
@@ -94,12 +94,25 @@ static string GetSupportDir()
 			return dir + Path.DirectorySeparatorChar;
 		}
 
-		public static string GameDir { get { return AppDomain.CurrentDomain.BaseDirectory; } }
+		public static string GameDir
+		{
+			get
+			{
+				var gameDir = AppDomain.CurrentDomain.BaseDirectory;
+
+				// BUG: In Game.cs we get GameDir with trailing DirectorySeparator, but not in PlatformTest.cs
+				if (!gameDir.EndsWith(Path.DirectorySeparatorChar.ToString()))
+				{
+					gameDir += Path.DirectorySeparatorChar;
+				}
```

Comment:
Alright, I'll check it out and add a new PR. There you can have visual studio use the file when compiling so you'll get errors for any violations.